section3/reflection.md

```diff
@@ -1,17 +1,37 @@
 ## Section 3 Reflection
 
 1. How different did your workflow feel this week, considering we asked you to follow the Pomodoro technique?
+- I feel like I was more productive, and less brain dead by the end of the day.
 
 2. Regarding the work you did around setting intentions in Step 1 of the Pomodoro technique - how did that go? Were you surprised by anything (did you find yourself way more focused than you realized, more distracted that you thought you'd be, estimating times accurately or totally off, etc)?
+- I was surprised at how quickly the time passed. And how hard it was to step away if I felt like I was in the middle of something.
 
 3. What is a Hash, and how is it different from an Array?
++ a Hash is a collection of data where each element of data is addressed by name
++ an Array is a container which hosts things from first to last. You can only use numbers to get data out of an Array
 
 4. In the space below, create a Hash stored to a variable named `pet_store`.  This hash should hold an inventory of items and the number of that item that you might find at a pet store.
 
+```ruby
+pet_store = {
+food: 1000,
+toys: 1000000000,
+good_animals: 400,
+bad_animals: 0
+}
+```
+
 5. Given the following `states = {"CO" => "Colorado", "IA" => "Iowa", "OK" => "Oklahoma"}`, how would you access the value `"Iowa"`?
 
+`states.values[1]`
```

Comment:
This works! Can you think of another, more direct way to access the value associated with a certain key in a hash?